### DISCUSSION TITLE
backdrop-blur-xx didn't work

### DISCUSSION THREAD

**Discussion by hezhi3f** [tailwindlabs/tailwindcss#12255] [Help]
![image](https://github.com/tailwindlabs/tailwindcss/assets/58511226/3c1f5dd9-24e2-41bd-b76a-f71dede9e745)
it didn't work
help me! why?

**Comment by alex-krasikau** [comment]
Hey @hezhi3f!

Works as expected for me - https://play.tailwindcss.com/PKfrfJt4Ly!

Could you please provide a reproduction example so we can take a closer look at it? It would also be beneficial to know which version of Tailwind CSS you are using, as well as the framework and Tailwind configuration.
